docs/APISocketOptions.md

```diff
@@ -841,31 +868,72 @@ missing packet still wasn't recovered, or wasn't conditionally dropped (see
-	`SRTO_INPUTBW` option set to 0 (automatic). Otherwise your stream will choke 
-	and break quickly at any rise in packet loss.*
 
-- ***To do: set-only; get should be supported.***
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_PACINGMODE
+
+| OptName               | Since | Binding | Type   | Units  | Default  | Range   | Dir | Entity |
+| --------------------- | ----- | ------- | ------ | ------ | -------- | ------- | --- | ------ |
+| `SRTO_PACINGMODE `    | 1.5.x | post    | `int`  |        |          | enum    | RW  | GSD    |
+
+Pacing control mode. This option is only applicable for live transmission mode.
+
+`SRTO_PACINGMODE` option defines the way of configuring the maximum allowed bandwidth `MAXBW`
+which limits the bandwidth usage by SRT and is used by live congestion control
+(LiveCC) module to calculate the minimum inter-sending time of consecutive packets.
+
+A combination of pacing control and live congestion control (LiveCC) modules controls SRT
+packets pacing based on the input rate and an overhead for packets retransmission
+in order to avoid congestion during fluctuations of the source bitrate.
+
+The following table shows the summary of pacing control modes, options need to be
+set (✓) or ignored (-) appropriately and formula for MAXBW calculation for a
+particular mode:
+
+| Mode / Option            | SRTO_MAXBW | SRTO_INPUTBW | SRT_OHEADBW | Formula for `MAXBW` Calculation                              |
+| ------------------------ | ---------- | ------------ | ----------- | ------------------------------------------------------------ |
+| `SRT_PACING_DEFMAXBW`    | -1         | -            | -           | `MAXBW` is set to the default value of 1Gbps                 |
+| `SRT_PACING_MAXBW`       | ✓          | -            | -           | `MAXBW` is set explicitly, in bytes per second               |
+| `SRT_PACING_INPUTBW`     | 0          | ✓            | ✓           | `MAXBW = INPUTBW * (1 + OHEADBW /100)`                       |
+| `SRT_PACING_ESTINPUTBW`  | 0          | 0            | ✓           | `MAXBW = ESTINPUTBW * (1 + OHEADBW /100)`                    |
+| `SRT_PACING_HYBRIDBW`    | 0          | ✓            | ✓           | `MAX_BW = max(INPUTBW, ESTINPUT_BW) * (1 + OHEADBW /100)`    |
+
+where `ESTINPUTBW` corresponds to the estimated (by SRT) SRT sender's input rate.
+
+See section (TODO: insert link to the new RFC draft once published) for the detailed description of pacing control modes.
+
+It's important to note that options [SRTO_MAXBW](#SRTO_MAXBW), [SRTO_INPUTBW](#SRTO_INPUTBW) and [SRT_OHEADBW](#SRT_OHEADBW) should be set first, right before setting the `SRTO_PACINGMODE` option. The order is important. In case the combination of `SRTO_MAXBW`, `SRTO_INPUTBW` and `SRTO_OHEADBW` options is configured wrong for a particular pacing control mode, SRT will throw an error (TODO: discuss what kind of error and put this information here).
```

Comment:
```suggestion
It's important to note that options [SRTO_MAXBW](#SRTO_MAXBW), [SRTO_INPUTBW](#SRTO_INPUTBW) and [SRTO_OHEADBW](#SRTO_OHEADBW) should be set first, right before setting the `SRTO_PACINGMODE` option. The order is important. In case the combination of `SRTO_MAXBW`, `SRTO_INPUTBW` and `SRTO_OHEADBW` options is configured wrong for a particular pacing control mode, SRT will throw an error (TODO: discuss what kind of error and put this information here).
```

Reply:
I don't agree with this. The PACING_MODE vs. MAXBW, INPUTBW, OHEADBW table was to compare manual setting vs. PACING setting. I thought the PR was taking care of settings in any order. PACING_MODE setting shall simplify, not complexify configuration. For example, setting SRT_PACING_ESTINPUTBW would force set MAXBW and INPUTBW to 0. only OHEADBW is required, before or after PACING_MODE. I think the approach of the PR is that the last setting is applied. For example setting a non-0 value in INPUTBW after setting SRT_PACING_ESTINPUTBW would switch the mode to SRT_PACING_INPUTBW. It could have been SRT_PACING_HYBRIDBW (even more logic). Particular care was taken to preserve backward compatibility and a neutral behavior for application not setting PACING_MODE. If error are thrown it should not be for applications working properly with previous releases.